### PR TITLE
Change GUI colors + update screenshots

### PR DESCRIPTION
**Pull request by realqijun**
#124 
#131

### CODE REVIEW

File: docs/images/Ui.png
Comment (uniqly) [inline docs/images/Ui.png:1]: Nice colours!

**Review by uniqly — APPROVED** [review]
LGTM

File: docs/images/Ui.png
Comment (blanklogic) [inline docs/images/Ui.png:1]: Love the new colour scheme!

File: src/main/java/seedu/internbuddy/ui/CompanyCard.java [hunk 1]
```diff
@@ -60,10 +60,13 @@ public class CompanyCard extends UiPart<Region> {
      */
     public CompanyCard(Company company, int displayedIndex) {
         super(FXML);
+
         assert displayedIndex > -1 : "Displayed index is less than zero";
         assert company != null : "Company is null";
 
         this.company = company;
+
+        // Initialize contents of CompanyCard
```
Comment (blanklogic) [inline src/main/java/seedu/internbuddy/ui/CompanyCard.java:69]: Good use of comments to help document code

**Pull request merged**